frontend/src/app/shared/components/fields/edit/field-types/multi-select-edit-field.component.ts

```diff
@@ -223,4 +226,30 @@ export class MultiSelectEditFieldComponent extends EditFieldComponent implements
       this.currentValueInvalid = !!this.schema.required;
     }
   }
+
+  /**
+   * For multi-select fields that are of type User, we want to show a hover card when hovering over users in the
+   * dropdown. For this to happen we must register a trigger target.
+   */
+  protected getHoverCardTriggerTarget() {
+    if (this.schema?.type === '[]User') {
+      return 'trigger';
+    }
+
+    return '';
+  }
+
+  /**
+   * For multi-select fields that are of type User, we want to show a hover card when hovering over users in the
+   * dropdown. For this to happen we must define a URL for the hover card.
+   */
+  protected getHoverCardUrl(item:any) {
```

Comment:
I believe the following type declaration should do the trick.
```suggestion
  protected getHoverCardUrl(item:{ id?:string }) {
```

Alternatively:
```typescript
protected getHoverCardUrl({ id }: { id?: string }) {
  if (id) {
  //...
  }
```

Reply:
It does, thank you! 👍🏻